ImageClassification/ViewControllers/CameraBoard.swift

```diff
@@ -173,33 +200,95 @@ extension Caboard: CameraFeedManagerDelegate {
 
 }
 
-extension Caboard: UITableViewDelegate, UITableViewDataSource {
-    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return prediction.count
-    }
 
-    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "cell")
-        cell?.textLabel?.text = prediction[indexPath.row]
-        cell?.textLabel?.textColor = .white
-        cell?.backgroundColor = .clear
-        return cell!
-    }
 
-    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        tableView.deselectRow(at: indexPath, animated: true)
-//        outputText.append(prediction[indexPath.row])
+extension CameraBoard {
 
-		DispatchQueue.main.async {
-//			print(self.outputText)
-			self.target?.insertText(self.prediction[indexPath.row])
-		}
+    func predictionStackSetup(){
+        addSubview(predictionStack)
+        predictionStack.translatesAutoresizingMaskIntoConstraints = false
+        predictionStack.leadingAnchor.constraint(equalTo: previewView.trailingAnchor, constant: 5).isActive = true
+        predictionStack.topAnchor.constraint(equalTo: topAnchor, constant: 5).isActive = true
+        predictionStack.bottomAnchor.constraint(equalTo: deleteButton.topAnchor, constant: -5).isActive = true
+        predictionStack.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -5).isActive = true
+        predictionStack.axis = .vertical
+        predictionStack.spacing = 5
+        predictionStack.distribution = .fillEqually
+        
+        var range = 0
+        if prediction.count > 3 {
+            range = prediction.count
+        } else {
+            range = predictionButton.count
+        }
+        var x = 0
+        while x < 3 {
+            predictionButton.append(UIButton())
+            predictionStack.addArrangedSubview(predictionButton[x])
+            predictionStack.translatesAutoresizingMaskIntoConstraints = false
+            predictionButton[x].titleLabel?.textAlignment = .left
+            predictionButton[x].setTitle("", for: .normal)
+            predictionButton[x].backgroundColor = #colorLiteral(red: 1.0, green: 1.0, blue: 1.0, alpha: 1.0).withAlphaComponent(0.2)
+            predictionButton[x].setTitleColor(#colorLiteral(red: 0, green: 0, blue: 0, alpha: 1), for: .normal)
+            predictionButton[x].addTarget(self, action: #selector(predictionButtonHoldDown(_:)), for: .touchDown)
+            predictionButton[x].addTarget(self, action: #selector(predictionButtonTapped(_:)), for: .touchUpInside)
+            predictionButton[x].addTarget(self, action: #selector(predictionButtonTapped(_:)), for: .touchDragExit)
+            x += 1
+        }
```

Comment:
I'm guessing this area updates the prediction buttons? try to explicitly say that with variable names and comments. or refactor code into functions